librz/analysis/meta.c

```diff
@@ -237,8 +237,8 @@ RZ_API const char *rz_meta_type_to_string(int type) {
 	return "# unknown meta # ";
 }
 
-RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64 size, int rad, PJ *pj, bool show_full) {
-	rz_return_if_fail(!(rad == 'j' && !pj)); // rad == 'j' => pj != NULL
+RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64 size, RzOutputMode mode, PJ *pj, bool show_full) {
+	rz_return_if_fail(!(mode == RZ_OUTPUT_MODE_JSON && !pj)); // rad == RZ_OUTPUT_MODE_JSON => pj != NULL
```

Comment:
I think it's safe to remove this comment completely, it is just the same as the code anyway.